### PR TITLE
Fix deploy-MSBuild after arcade change

### PR DESCRIPTION
**Pull request by Forgind**
Arcade change didn't take into account Deploy-MSBuild, and it failed with a Microsoft.NET.StringTools.dll does not exist error. That was actually very fortunate (thanks ladipro!) since it otherwise would have silently accepted looking into the wrong folder (from a previous bootstrap build) and deploying something that hadn't actually been updated, which would have been extremely confusing.

This only fixes the instance I found, but I would recommend someone look for other similar instances.

Also, assuming this is eventually merged, @BenVillalobos for awareness, since it would have to be reverted also if we revert the arcade change.

### CODE REVIEW

**Review by BenVillalobos — APPROVED** [review]
LGTM. We should wait until arcade is fully merged before getting this in.

**Comment by Lodewijk1** [comment]
Master Joda

**Pull request merged**
